src/components/SelectionListRadio/BaseSelectionListRadio.js

```diff
@@ -0,0 +1,378 @@
+            } else {
+                textInputRef.current.focus();
+            }
+        }
+
+        return () => {
+            if (!focusTimeout) {
+                return;
+            }
+            clearTimeout(focusTimeout);
+        };
+    }, [props.shouldDelayFocus, shouldShowTextInput]);
+
+    useEffect(() => {
+        scrollToIndex(focusedIndex);
+    }, [focusedIndex, scrollToIndex]);
+
+    useEffect(() => {
+        const enterConfig = CONST.KEYBOARD_SHORTCUTS.ENTER;
+        const unsubscribeEnter = KeyboardShortcut.subscribe(
+            enterConfig.shortcutKey,
+            () => {
+                const focusedOption = flattenedSections.allOptions[focusedIndex];
+
+                if (!focusedOption) {
+                    return;
+                }
+
+                props.onSelectRow(focusedOption);
+            },
+            enterConfig.descriptionKey,
+            enterConfig.modifiers,
+            true,
+            () => !flattenedSections.allOptions[focusedIndex],
+        );
+
+        const CTRLEnterConfig = CONST.KEYBOARD_SHORTCUTS.CTRL_ENTER;
+        const unsubscribeCTRLEnter = KeyboardShortcut.subscribe(
+            CTRLEnterConfig.shortcutKey,
+            () => {
+                const focusedOption = flattenedSections.allOptions[focusedIndex];
+
+                if (!focusedOption) {
+                    return;
+                }
+
+                props.onSelectRow(focusedOption);
+            },
+            CTRLEnterConfig.descriptionKey,
+            CTRLEnterConfig.modifiers,
+            true,
+        );
+
+        return () => {
+            if (unsubscribeEnter) {
+                unsubscribeEnter();
+            }
+
+            if (unsubscribeCTRLEnter) {
+                unsubscribeCTRLEnter();
+            }
+        };
+    }, [flattenedSections.allOptions, focusedIndex, props]);
+
+    return (
+        <ArrowKeyFocusManager
+            disabledIndexes={flattenedSections.disabledOptionsIndexes}
+            focusedIndex={focusedIndex}
+            maxIndex={flattenedSections.allOptions.length - 1}
+            onFocusedIndexChanged={setFocusedIndex}
+        >
+            <View style={[styles.flex1]}>
+                {shouldShowTextInput && (
+                    <View style={[styles.ph5, styles.pv5]}>
+                        <TextInput
+                            ref={textInputRef}
+                            label={props.textInputLabel}
+                            value={props.textInputValue}
+                            placeholder={props.textInputPlaceholder}
+                            maxLength={props.textInputMaxLength}
+                            onChangeText={props.onChangeText}
+                            keyboardType={props.keyboardType}
+                            selectTextOnFocus
+                        />
+                    </View>
+                )}
+
```

Comment:
```suggestion
```

Reply:
I think adding an empty line at the end is forced by prettier